src/library/scala/collection/convert/Wrappers.scala

```diff
@@ -411,24 +411,21 @@ private[collection] trait Wrappers {
 
     override def size = underlying.size
 
-    def get(k: String) = {
-      val v = underlying get k
-      if (v != null) Some(v.asInstanceOf[String]) else None
+    def get(k: String): Option[String] = {
+      Option((underlying get k).asInstanceOf[String])
```

Comment:
Function calling would, if not optimized out by the JVM, probably add about 5% to the execution time of the code, and the JVM is pretty good at optimizing out such things.  But I don't see much reason to change clear, working code into different clear, working code that might be slower without a demonstration that it really isn't.  Does the change let you sneak under the magic JVM inlining budget, for example?